ui/src/app/service/pipeline/pipeline.service.ts

```diff
@@ -23,8 +23,13 @@ export class PipelineService {
      * @param key Project unique key
      * @param pipName Pipeline Name
      */
-    getPipeline(key: string, pipName: string): Observable<Pipeline> {
-        return this._http.get('/project/' + key + '/pipeline/' + pipName).map(res => res.json());
+    getPipeline(key: string, pipName: string, withApplications: boolean = false): Observable<Pipeline> {
```

Comment:
At the moment withApplications is always true. I have put this parameter because it isn"t a mandatory parameter so by default it is false in order to make less query if we doesn't care. If you want I can put this param to true by default